src/daemon.py

```diff
@@ -67,6 +67,14 @@ def __search_task(self, job_id: JobId) -> None:
         """
         logging.info("Parsing...")
 
+        result = self.ursa.topology()
```

Comment:
why move `.topology()` here? this is necessary only for new jobs, so this line should stay where it was.

Reply:
I use it to check if any datasets are indexed so this is needed for whole __search_task function. But if you want I can change it but then we will need to do the same operation self.ursa.topology() twice.